mesos_slave/datadog_checks/mesos_slave/mesos_slave.py

```diff
@@ -113,139 +116,117 @@ def __init__(self, name, init_config, instances):
             )
             self.http.options['timeout'] = (timeout, timeout)
 
-    def _get_json(self, url, failure_expected=False, tags=None):
-        tags = tags + ["url:%s" % url] if tags else ["url:%s" % url]
-        msg = None
-        status = None
-        timeout = self.http.options['timeout']
-        response = None
+    def check(self, instance):
+        if 'url' not in instance:
+            raise Exception('Mesos instance missing "url" value.')
```

Comment:
```suggestion
            raise ConfigurationError('Mesos instance missing "url" value.')
```
Can be moved to constructor